backend/sirius-web-diagrams/src/main/java/org/eclipse/sirius/web/diagrams/Diagram.java

```diff
@@ -50,6 +50,8 @@
 
     private Size size;
 
+    private boolean autoLayout;
```

Comment:
Why provide this info on the GraphQL API of the diagram POJO while it already exists on the GraphQL API of DiagraDescription?